tests/main/security-seccomp/task.yaml

```diff
@@ -0,0 +1,107 @@
+summary: Check basic seccomp rules
+
+details: |
+    This test verifies that:
+    .
+    - absence of a syscall is denied
+    - use of a bare syscall (ie, no arguments) is allowed
+    - use of a syscall with arg filtering is allowed with matching arguments
+    - use of a syscall with arg filtering is denied with unmatching arguments
+    .
+    We choose the setpriority syscall for these tests since it is available on
+    all architectures and can be easily used to test all of the above. As part of
+    its testing, this test will modify the snap's seccomp filter and recompile
+    it. On strict mode systems, it will also adjust the apparmor profile to allow
+    'capability nice' so it doesn't interfere with anything.
+
+# IMPORTANT: adding a system here is an explicit statement that snapd doesn't
+# care that the system has broken seccomp mediation. This is unlikely what you
+# want. If adding, please add why and any references to supporting context.
+# See:
+# https://forum.snapcraft.io/t/disabling-seccomp-sandbox-where-a-buggy-golang-seccomp-is-used/11054
+#
+# core systems do not have gcc so exclude them. This is not ideal but the core
+# systems are built from the Ubuntu archive using kernels branched from distro
+# kernels and we test Ubuntu classic distro here.
+systems: [-ubuntu-core-*]
```

Comment:
If we make the test tool a snap and publish it, then the test cold run on Core systems too.

Reply:
Done